python/openmldb_tool/diagnostic_tool/table_checker.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright 2021 4Paradigm
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+import termplotlib as tpl
+
+from .connector import Connector
+
+
+class TableChecker:
+    def __init__(self, conn: Connector):
+        self.conn = conn
+
+    def check_distribution(self, dbs: list):
+        ns_leader = self._get_nameserver()
+        url = f"http://{ns_leader}/NameServer/ShowTable"
+        res = requests.get(url, json={"show_all": True})
+        tables = res.json()["table_info"]
+
+        exist_dbs = [db[0] for db in self.conn.execfetch("SHOW DATABASES")]
+        if "all" in dbs:
```

Comment:
We can't prevent a database named 'all'. So it should be:
1. dbs is none, print all dbs
2. dbs is not empty, print specified dbs
`--db` should be not required.